nextflow.config

```diff
@@ -3,108 +3,98 @@
  *  nf-core/nascent Nextflow config file
  * -------------------------------------------------
  * Default config options for all environments.
- * Cluster-specific config options should be saved
- * in the conf folder and imported under a profile
- * name here.
  */
 
 // Global default params, used in configs
 params {
-
-  // Container slug. Stable releases should specify release tag!
-  //   Developmental code should specify :latest
-  container = 'ignaciot/nascent:latest'
-
-  // Workflow flags
-  //reads = "data/*{R1,R2}*.fastq"
-  //fastqs= "data/*{R1,R2}*.fastq"
-  //sras= "data/*{R1,R2}*.sra"
-  outdir = './results'
     
-// Run arguments
+  // Run arguments
   workdir = false
   clusterOptions = false
   flip = false
-  email=false
-  help = false
-  igenomes_base = "./iGenomes"
-  tracedir = "${params.outdir}/pipeline_info"
-//  nf_required_version = nf_required_version
-  outdir = './results'    
-  pairedEnd = false
   saveAllfq = false
   savefq = false
   saveTrim = false
   skipMultiQC = false
   threadfqdump = false
-//  version = version    
+
+  //TODO IGNACIO CHECK 
```

Comment:
Please check that these default values are just fine for most users and document them :)

Reply:
Properly documented them here an in the usage docs.